Cargo.toml

```diff
@@ -125,6 +125,7 @@ winres = "0.1.12"
 
 [dev-dependencies]
 mockall = "0.11"
+temp-env = "0.3.2"
```

Comment:
Ah alright
I'm thinking a few things then:
* A new method `Context#get_config_path`
* Would need `Context#new_with_env` for testing since the init of `Context` uses STARSHIP_CONFIG / HOME on `new`
* There is `config.rs#get_config_path` and `configure.rs#get_config_path`. They look practically identical to me, I think they can be consolidated to one definition in a new method `Context#get_config_path`?

Code in `master` is like this:

```mermaid
classDiagram
`configure.rs#print_configuration` --> `config.rs#StarshipConfig` : in initialize()
`config.rs#StarshipConfig` --> env : config.rs#get_config_path fetches STARSHIP_CONFIG
`context.rs#Context` *-- `config.rs#StarshipConfig` : in new()
```

I think these ideas are ok if the idea is to have all env mocking in `Context`, what do you think?